### PR TITLE
feat(DEV-11544): migrate payables to qraft

### PR DESCRIPTION
**Pull request by saladNights**
Payables have left the dependency from `sdk-api` for MoniuteSDK provider and counterparts-related hook and type.

`sdk-api` dependency was entirely removed from the payables mock.

### CODE REVIEW

File: packages/sdk-react/src/api/client.ts [hunk 1]
```diff
@@ -54,6 +53,6 @@ export const createAPIClient = ({
 };
 
 export const isMoniteEntityIdPath = (path: string) =>
-  /^\/(?!auth|entities|entity_users\/me|entity_users\/my_entity|entity_users\/my_role|events|mail_templates|webhook_subscriptions|webhook_settings|receivables\/variables|settings|files|mailbox_domains|payable_purchase_orders|frontend|internal)\b/.test(
+  /^\/(?!auth|entities|entity_users\/my_entity|events|mail_templates|webhook_subscriptions|webhook_settings|receivables\/variables|settings|files|mailbox_domains|payable_purchase_orders|frontend|internal)\b/.test(
```
Comment (radist2s) [inline packages/sdk-react/src/api/client.ts:56]: Let's not send the Headers which are not needed. I suppose these RegExp parts are still needed:
```
/me|entity_users\/my_entity|entity_users\/my_role
```

File: .changeset/flat-parrots-care.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+'@monite/sdk-react': minor
+'@monite/sdk-api': minor
+---
+
+feat(DEV-11544): migrate Payables to qraft
```
[the remark this answers is not in the document]
Comment (saladNights) [reply .changeset/flat-parrots-care.md]: fixed

File: packages/sdk-react/src/components/payables/PayableDetails/PayableDetailsAttachFile/PayableDetailsAttachFile.tsx [hunk 3]
```diff
@@ -39,10 +41,31 @@ export const PayableDetailsAttachFile = ({
   payableId,
 }: PayableDetailsAttachFileProps) => {
   const { i18n } = useLingui();
+  const { api } = useMoniteContext();
+  const queryClient = useQueryClient();
```
[the remark this answers is not in the document]
Comment (saladNights) [reply packages/sdk-react/src/components/payables/PayableDetails/PayableDetailsAttachFile/PayableDetailsAttachFile.tsx]: thanks!

File: packages/sdk-react/src/api/client.ts (hunk 1 above)
Comment (saladNights) [reply packages/sdk-react/src/api/client.ts:57]: agreed to fix it in https://monite.atlassian.net/browse/DEV-11719

**Pull request merged**